packages/html/ds/src/button/button.stories.ts

```diff
@@ -74,24 +74,69 @@ export const Default: Story = {
 export const AllVariants: Story = {
   //@ts-expect-error Render function returns raw HTML string, not a React component
   render: () => `
-    <div class="gi-flex gi-flex-col gi-gap-4">
-      <div class="gi-flex gi-gap-4">
-        <button class="gi-btn gi-btn-primary gi-btn-regular">Primary</button>
-        <button class="gi-btn gi-btn-secondary gi-btn-regular">Secondary</button>
-        <button class="gi-btn gi-btn-flat gi-btn-regular">Flat</button>
-      </div>
-      <div class="gi-flex gi-gap-4">
-        <button class="gi-btn gi-btn-primary-dark gi-btn-regular">Primary Dark</button>
-        <button class="gi-btn gi-btn-secondary-dark gi-btn-regular">Secondary Dark</button>
-        <button class="gi-btn gi-btn-flat-dark gi-btn-regular">Flat Dark</button>
-      </div>
-      <div class="gi-flex gi-gap-4 gi-bg-black gi-p-4 gi-w-fit">
-        <button class="gi-btn gi-btn-primary-light gi-btn-regular">Primary Light</button>
-        <button class="gi-btn gi-btn-secondary-light gi-btn-regular">Secondary Light</button>
-        <button class="gi-btn gi-btn-flat-light gi-btn-regular">Flat Light</button>
-      </div>
+  <div class="gi-flex gi-flex-col gi-gap-4">
```

Comment:
Great work on the PR. Not for this PR but I was thinking going forward we should investigate if there's another way to render the stories in the macro because doing it like this we open ourselves to the stories not matching the component. For example if in the future we change something within the macro, we also have to manually change it here.